backend/alembic/versions/20260409_a1b2c3d4e5f6_add_inbox_ai_fields.py

```
@@ -0,0 +1,183 @@
+    if dialect == "mysql":
+        try:
+            op.execute(
+                sa.text(
+                    "ALTER TABLE queue_messages MODIFY COLUMN status "
+                    "ENUM('unread','read','processing','processed','archived','failed') "
+                    "NOT NULL DEFAULT 'unread'"
+                )
+            )
+        except Exception:
+            # Column may already have the 'failed' value
+            pass
```

Ruff (0.15.9)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)